Casks/l/lg-onscreen-control.rb

```diff
@@ -8,8 +8,25 @@
   desc "Displays all connected LG monitor information"
   homepage "https://www.lg.com/us/support/monitors"
 
+  # There is no page available specifically for the software
+  # so we return the downloads from one of the popular products
   livecheck do
-    skip "No version information available"
+    url "https://www.lg.com/us/support/product/lg-27GN950-B.AUS"
+    regex(/Mac[._-]OSC[._-]v?(\d+(?:\.\d+)+)\.zip/)
+    strategy :page_match do |page, regex|
+      json_string = page&.[](/NEXT[._-]DATA[^>]*>\s*([^<]+)\s*</i, 1)
```

Comment:
```suggestion
      json_string = page[/NEXT[._-]DATA[^>]*>\s*([^<]+)\s*</i, 1]
```

Due to how strategies handle `content`, `page` will always exist here if execution reaches the `strategy` block.